api/src/main/java/org/openmrs/module/pihcore/identifier/peru/ConfigurePeruIdGenerators.java

```diff
@@ -0,0 +1,59 @@
+package org.openmrs.module.pihcore.identifier.peru;
+
+import org.openmrs.Location;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.module.idgen.AutoGenerationOption;
+import org.openmrs.module.idgen.SequentialIdentifierGenerator;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+import org.openmrs.module.metadatadeploy.MetadataUtils;
+import org.openmrs.module.pihcore.config.Config;
+import org.openmrs.module.pihcore.metadata.peru.PeruPatientIdentifierTypes;
+
+public class ConfigurePeruIdGenerators {
```

Comment:
tangentially, we should make sure we remember to let the SES team know that this is just a default format we came up with, and there are a lot of custom options available we could use.

Reply:
I'm going to swap this out with a CSV as soon as I implement the Iniz PIT loader